runtime/lua/vim/autoread.lua

```diff
@@ -0,0 +1,165 @@
+-- Create a file watcher, each watcher is identified by the buffer number of the file it is watching.
+
+local uv = vim.loop
+
+local Watcher = {}
```

Comment:
nit: 
```suggestion
local M = {}
```

Reply:
Out of curiousity, why is `M = {}` ubiquitous in the codebase?